OpenRA.Mods.RA/Player/AllyRepair.cs

```diff
@@ -8,10 +8,10 @@
  */
 #endregion
 
+using OpenRA.Mods.RA.Traits;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.RA.Traits
-
+namespace OpenRA.Mods.RA
```

Comment:
This should stay as `namespace OpenRA.Mods.RA.Traits`.
Looks like fallout from a rebase conflict.